### PR TITLE
Add blog post about differences between PayPal extensions

### PR DESCRIPTION
**Pull request by seand7565**

### CODE REVIEW

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/2020/09/02/solidus-paypal-commerce-platform/cover.jpg
+description: >
+  PayPal has been working with Solidus in a big way over the last year, find out what's new in the world of payment gateways, and which one makes more sense for your store.
+---
+
+We're excited to share some news with you. As you are probably aware, Solidus has been working with PayPal recently. They've been the gold sponsor for our last two conferences - and more recently, they've helped us revitalize Solidus PayPal Braintree and release the new Solidus PayPal Commerce Platform extension.
+
+We're excited to announce that Solidus PayPal Commerce Platform has just released version 0.2.1, and that Solidus PayPal Braintree has just released version 1.0. Both of these extensions are ready to go, all _you_ need to do is find out which one is right for you.
```
Comment (kennyadsl) [inline source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:13]: `Solidus PayPal Braintree has just released version 1.0`

should be 

`Solidus PayPal Braintree has just released version 1.0.0`

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+If you're already on this extension, the most important changes are that the extension now works with Rails 6, and you can now pass placeholder text or styling instructions to the Braintree hosted fields, making designing around the credit card inputs a bit easier.
+
+So why use this extension over Solidus PayPal Commerce Platform? Well first, it offers more options for your consumers. Through Braintree, you're able to accept credit cards with 3DSecure enabled, and Apple Pay as well. Through hosted fields, you're also given a bit more control over the checkout experience. 
+
+<img src="/blog/2020/11/11/using-paypal-with-solidus/placeholder.png" />
+
+The downside of this extension is that it requires a lot more setup than Solidus PayPal Commerce Platform. The initial setup can take a while, and can be confusing. If you want to use PayPals credit messaging, you'll have to change your settings around in a particular way to allow that. If you want to utilize the cart checkout, you'll need to add the cart checkout partial.
```
Comment (kennyadsl) [inline source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:35]: ` and can be confusing.` I'd say that uses a developer-friendly language and install procedure instead.

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Solidus PayPal Commerce Platform
```
Comment (kennyadsl) [inline source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:37]: We can probably also say that with SPC we are delegating all the responsibilities around paying with Credit Cards (like fraud detections) to PayPal, while with Braintree should be up to the store owner to take care of that.

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: Using PayPal with Solidus
+date: 2020-11-11
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/2020/09/02/solidus-paypal-commerce-platform/cover.jpg
```
Comment (kennyadsl) [inline source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:6]: I think it's better to copy this image over here as well. If we remove or change that blog post path this one will break along.

**Review by kennyadsl — COMMENTED** [review]
Thanks Sean, I left some comments. Also, I can't see the post in the preview, is that expected?

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+We're excited to share some news with you. As you are probably aware, Solidus has been working with PayPal recently. They've been the gold sponsor for our last two conferences - and more recently, they've helped us revitalize Solidus PayPal Braintree and release the new Solidus PayPal Commerce Platform extension.
+
+We're excited to announce that Solidus PayPal Commerce Platform has just released version 0.2.1, and that Solidus PayPal Braintree has just released version 1.0. Both of these extensions are ready to go, all _you_ need to do is find out which one is right for you.
+
+### Two Extensions, Two Paths
+
+Both extensions bring the magic PayPal buttons into your store, but they operate in very different ways. Below is a quick breakdown of what to expect from each extension.
+
+Before we begin though, it's important to note that both of these extensions use the updated PayPal SDK. This will allow your consumers to checkout using PayPals new financing options, as well as use any payment methods that might be local for them - for instance, your customers from Italy will be able to check out with MyBank, an Italian payment solution.
```
Comment (mtylty) [inline source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:19]: Paypals -> Paypal's

**Review by mtylty — CHANGES_REQUESTED** [review]

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown (hunk 2 above)
Comment (seand7565) [reply source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:35]: Maybe we can split the difference with `and is geared more towards developers`? That section is supposed to be a list of reasons why you might _not_ want to use that extension, so I want to frame in that way. But I do think `can be confusing` is too negative too, so we should definitely change it.

File: source/blog/2020-11-11-using-paypal-with-solidus.html.markdown (hunk 4 above)
Comment (seand7565) [reply source/blog/2020-11-11-using-paypal-with-solidus.html.markdown:6]: Yeah, good idea, thanks!

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**